packages/test-helpers/constants.js

```diff
@@ -0,0 +1,2 @@
+export const ANY_ADDRESS = '0xffffffffffffffffffffffffffffffffffffffff'
```

Comment:
Include the two constant used very frequently. Do you think there are others that make sense?